FRAuth/FRAuth/Auth/Node.swift

```diff
@@ -268,7 +268,7 @@ public class Node: NSObject {
     func buildAuthServiceRequest() -> Request {
         
         var header: [String: String] = [:]
-        header[OpenAM.xRequestedWith] = OpenAM.xmlHTTPRequest
+        header[OpenAM.acceptAPIVersion] = OpenAM.apiResource21 + "," + OpenAM.apiProtocol10
```

Comment:
I would recommend add inline comments regarding this supported APIs in the SDK.

Reply:
Added